78 life cycle management (#94)
boucadair/attachment-circuit-model: draft-boro-opsawg-teas-attachment-circuit.md

The same customer site (CE, NF, etc.) can terminate one or multiple bearers; each of them uniquely identified by a reference that is assigned by the network provider. These bearers can terminate on the same or distinct network nodes. CEs that terminate multiple bearers are called multi-homed CEs.

A bearer can be created, modified or discovereed from the network. each case offer serverl possibilities:

> 'Greenfield Creation':

Reviewer: ```suggestion
'Greenfield creation':
```

: In this scenario, bearers are created from scratch based on specific requests made to the network controller or management system. This method offers fine-grained control over bearer attributes and placement constraints ('placement-constraints'). Allows providers to tailor bearer creation to meet customer-specific needs and offers fine-grained control for tailoring bearer provisioning to meet customer-specific needs.

> 'Auto-Discovery using Network Protocols':

Reviewer: ```suggestion
'Auto-discovery using network protocols':
```

> : Bearers can also be established using network protocols, such as Link Layer Discovery Protocol (LLDP), which enables devices to automatically discover and connect to available network resources. This method simplifies the provisioning process by allowing devices to discover and establish bearers autonomously, reducing manual configuration efforts. Well-suited for dynamic environments with changing network topologies.

Reviewer: ```suggestion
: Devices can use specific protocols (e.g., Link Layer Discovery Protocol (LLDP)) to automatically discover and connect to available network resources. A network controller can use such reported information to expose discovered bearers from the network using the same bearer structure.
```

'status':
: Used to track the overall status of a given bearer. Both operational and administrative status are maintained together with a timestamp.

'adnmin-status':
: The "admin-status" attribute is typically configured by a network administrator or operator to determine whether the network element or service is enabled, disabled, or subjected to additional testing or pre-deployment checks. These additional options, such as 'admin-testing' and 'admin-pre-deployment', provide the operators the flexibility to conduct additional validations on the bearer before deploying services over that connection.
: It's important to note that the "admin-status" attribute should remain independent of the "oper-status". In other words, the intended administrative state, whether "admin-up" or "admin-testing", should not be influenced by the current operational state. If the bearer is administratively set to 'admin-down', it is expected that the bearer will also be operationally 'op-down' as a result of this administrative decision.

'oper-status':

> : The "oper-status" of a service reflects its operational state as observed at a particular bearer. As a bearer can contain multiple services, the operational status should only reflect the status of the bearer connection. To obtain network-level service status, specific network models such as those in {{Section 7.3 !RFC9182}}  or {{Section 7.3 !RFC9291}} should be consulted.

Reviewer: ```suggestion
: The "oper-status" of a service reflects its operational state as observed at a particular bearer. As a bearer can contain multiple services, the operational status should only reflect the status of the bearer connection. To obtain network-level service status, specific network models such as those in {{Section 7.3 of !RFC9182}}  or {{Section 7.3 of !RFC9291}} should be consulted.
```

> : The "oper-status" of a service reflects its operational state as observed at a particular bearer. As a berer can contain multiple services the operational status should only reflect the status of the bearer connection. To obtain network-level service status, specific network models, such as those outlined in {{!RFC9182}} Section 7.3 or {{!RFC9291}} Section 7.3, should be consulted.

Reviewer: ```suggestion
: The "oper-status" of a service reflects its operational state as observed at a particular bearer. As a bearer can contain multiple services, the operational status should only reflect the status of the bearer connection. To obtain network-level service status, specific network models such as those in {{Section 7.3 !RFC9182}}  or {{Section 7.3 !RFC9291}} should be consulted.
```

'Assessing Bearer Delivery Status':
: To assess the service delivery status for a given bearer comprehensively, it's recommended to consider both administrative and operational service statuses in conjunction. This holistic approach  allows a network controller or operator to identify anomalies effectively.

'Anomaly Detection':

> : For instance, when a beraer is administratively enabled but the "operational-status" of that bearer is reported as "down" it should be expected that the services's transported for thar bearer "oper-status" is also "down." If these statuses differ, it serves as a trigger to detect an anomaly.

Reviewer: ```suggestion
: For instance, when a bearer is administratively enabled but the "operational-status" of that bearer is reported as "down" it should be expected that the "oper-status" of the services transported for that bearer is also "down." If these statuses differ, it serves as a trigger to detect an anomaly.
```
